content/en/blog/_posts/2022-04-28-cni-version-related-breaking-change.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: blog
+title: "Breaking changes related to CNI version in popular container runtimes"
+date: 2022-04-28T12:00:00+0000
+slug: cni-version-support-changes-container-runtimes
+---
+
+**Authors:** Mike Brown (IBM), Rolfe Dlugy-Hegwer (Red Hat), and others.
+
+---
+
+_TL;DR: Before you upgrade to Kubernetes 1.24, if you use a CNI network plugin
+that is set up to use Container Network Interface (CNI) versions earlier than
+v1.0.0 and you plan to use any of the following container runtimes, you must
+specify the CNI version. Otherwise, your network will break._
+
+This issue applies to the following container runtimes:
+- **containerd** v1.6.0 and later
```

Comment:
confirm with @mikebrow exactly which versions of containerd are affected -- I think this range now has an upper bound.